config/build_image.sh

```diff
@@ -52,7 +52,13 @@ go install go.uber.org/mock/mockgen@${MOCKGEN_VERSION}
 # yq
 ####
```

Comment:
```suggestion
# Fetch the binary directly to avoid cache artifacts in final image. More info in PR description.
```

Reply:
resolving this as this is already standard https://github.com/openshift/boilerplate/blob/571b278a49528f3512e754ec9041668c0fb62254/config/build_image.sh#L74-L98